javascript/javascript_sdk/src/get_data.js

```diff
@@ -0,0 +1,85 @@
+const google_protobuf_timestamp_pb = require('google-protobuf/google/protobuf/timestamp_pb.js');
+const pointMsg = require('./../js_proto/sensr_proto/point_cloud_pb');
+
+const get_zone_data = (client) => {
+    client.listenToObjectUpdate(null, response => {
+      let zone_lists = response.array;
+      if(zone_lists[0][0] !== undefined){
```

Comment:
Can you use field name instead of index?
Javascript protobuf seems like providing to access a specific field.
`zone_lists[0][0]` -> `zone_lists.getZoneList().getTimestamp()`
https://developers.google.com/protocol-buffers/docs/reference/javascript-generated